packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -360,14 +367,23 @@ class Scaffold extends StatefulWidget {
   final List<Widget> persistentFooterButtons;
 
   /// A panel displayed to the side of the [body], often hidden on mobile
-  /// devices.
+  /// devices. Swipes in from left-to-right
   ///
   /// In the uncommon case that you wish to open the drawer manually, use the
   /// [ScaffoldState.openDrawer] function.
   ///
   /// Typically a [Drawer].
   final Widget drawer;
 
+  /// A panel displayed to the side of the [body], often hidden on mobile
+  /// devices. Swipes in from right-to-left
```

Comment:
This last sentence is not accurate in RTL languages.

Reply:
Fixed